### DISCUSSION TITLE
PlaceHolder not registering on Papi 2.10.8

### DISCUSSION THREAD

**Discussion by LoSCraft-Network** [GriefPrevention/GriefPrevention#923] [Q&A] [answered]
### What happened:
Error on registering placeholder and placeholders not working

### What was expected:
Placeholders to hook

### Steps to reproduce:
1. Update to lastest PAPI (2.10.8)

### Server and GriefPrevention version:
___This can be found by running `/version` and `/version GriefPrevention` on your server.___
Paste of `/version`: git-Tuinity-"650532a"
Paste of `/version GriefPrevention`: 16.14.0

### Stack trace/error or server log
https://pastebin.com/4fqtTZhK

**Comment by polarassassin** [comment]
I have the same issue, the recent PAPI update changed some things and Grief prevention api expansion needs updating

**Comment by Jikoo** [comment, marked as answer]
GP does not provide a PAPI hook, PAPI offers a GP hook. Try re-downloading the hook via their system, and if you still encounter the issue contact PAPI.

**Comment by Kqliber** [comment]
please try this expansion, from testing so far it works. https://github.com/Kqliber/Expansions/releases/tag/2.0
